src/plugins/http/inc/httpParser.h

```diff
@@ -100,6 +100,7 @@ typedef enum HTTP_PARSER_STATE {
   HTTP_PARSER_CHUNK,
   HTTP_PARSER_END,
   HTTP_PARSER_ERROR,
+  HTTP_PARSER_OPTIONAL_SP,
```

Comment:
It's better to remove the last comma.

Reply:
OK, but there are many places in tdengine that end with commas.
Should we develop a common standard and introduce testing tools

Reply:
Testing tools, plugins, patches and documents are all appreciated:)